package.json

```diff
@@ -39,7 +39,7 @@
     "tsconfig": "workspace:*",
     "tsup": "^7.2.0",
     "turbo": "^1.10.13",
-    "typescript": "^5.2.2",
+    "typescript": "^5.3.2",
```

Comment:
Should this be 5.4.2?

Reply:
Turns out PNPM already did that under the covers but explicit is better